frontend/src/js/components/settings/organization/Billing.tsx

```diff
@@ -172,7 +213,46 @@ export const Billing = () => {
             </div>
           }
         />
-        {!organization.trial && !isEnterprise && <OrganizationPaymentSettings />}
+        {billing && changeBilling && (
+          <PlanExpanded
+            isEdit
+            onCloseClick={() => setChangeBilling(false)}
+            currentBillingProfile={{ address: { ...billing.address }, name: billing.name, email: billing.email }}
+            card={card}
+          />
+        )}
+        <div className={classes.billingSection}>
+          <div className="flexbox center-aligned">
+            <div className={classes.emailTooltip}>
+              <b>Billing details</b>
+            </div>
+            {!isEnterprise && billing && billing.address && (
```

Comment:
`billing.address` seems to not work with the API response & the type definition shows the data duplication - OTOH maybe the backend response type has changed again 🤷‍♂️

Reply:
Yes, there there where issue with backend, https://github.com/mendersoftware/mender-server-enterprise/pull/214
It should work with image from this PR